_episodes_rmd/05-s4.Rmd

```diff
@@ -158,6 +158,14 @@ df1 <- as.data.frame(DF1)
 df1
 ```
 
+Vice versa, we can also convert `data.frame` objects to `DataFrame` using the `as()` function.
+
+```{r}
+as(df1, "DataFrame")
+```
+
+and vice versa
```

Comment:
Could you please just clean up this bit? It doesn't look like a full sentence. And you just used 'vice versa' a line earlier.

Otherwise it all looks great, I look forward to merge as soon as this is cleaned up.

Thanks for spotting the typos!

Reply:
Oops, not sure what happened there. Should be fixed now!